### PR TITLE
added text input component

### PR DESCRIPTION
**Pull request by MeenuKumari28**
added text input component and fixed button size
width-input

### CODE REVIEW

**Comment by amitu** [comment]
@MeenuKumari28 show screenshot of how it looks.

**Comment by MeenuKumari28** [comment]
> @MeenuKumari28 show screenshot of how it looks.

input

**Comment by amitu** [comment]
The icons-1, 2, rename them to icon-start and icon-end. Also input fields should have error states, with an error message. Also instead of `radius: rounded`, make it `rounded: true` (with the default value of `false`). Actually I am not sure if rounded: true is looking that good. Do we really need it @yashveer29?

Also I prefer this behaviour, as you see on acko.com:

https://github.com/fastn-community/design-system/assets/58662/441eb270-db5d-4043-a297-5a892749b0ba

**Comment by MeenuKumari28** [comment]
> The icons-1, 2, rename them to icon-start and icon-end. Also input fields should have error states, with an error message. Also instead of `radius: rounded`, make it `rounded: true` (with the default value of `false`). Actually I am not sure if rounded: true is looking that good. Do we really need it @yashveer29?
> 
> Also I prefer this behaviour, as you see on acko.com:
> 
>  Screen.Recording.2024-01-18.at.1.18.19.PM.mov

@amitu yes we have error state also with error message.
error-input

**Comment by amitu** [comment]
@MeenuKumari28 update the PR description with the screenshot, and keep updating that place only every time you change things.

File: form/input.ftd [hunk 1]
```diff
@@ -17,7 +17,6 @@
 
 -- constant integer none: 0
 -- constant integer curved: 1
--- constant integer round: 2
 
 -- end: input-radius
```
Comment (amitu) [inline form/input.ftd:21]: Are we still using this type? If not lets delete it.

File: form/input.ftd [hunk 2]
```diff
@@ -50,12 +50,12 @@ inset: $ds.spaces.inset-square.small
 	if: { text-input.label != NULL }
 	
 	-- ds.row:
+	width.fixed.px: 360
```
Comment (amitu) [inline form/input.ftd:53]: Hard coding this is not good.

**Comment by yashveer29** [comment]
> The icons-1, 2, rename them to icon-start and icon-end. Also input fields should have error states, with an error message. Also instead of `radius: rounded`, make it `rounded: true` (with the default value of `false`). Actually I am not sure if rounded: true is looking that good. Do we really need it @yashveer29?
> 
> Also I prefer this behaviour, as you see on acko.com:
> 
>  Screen.Recording.2024-01-18.at.1.18.19.PM.mov

@amitu @MeenuKumari28 We can not make it rounded true/false because we have 2 types of roundness of the fields.

**Pull request merged**